server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -248,6 +249,15 @@ private PlatformConfig getPlatformConfig(String platform) throws ExtenderExcepti
         return platformConfig;
     }
 
+    private String getPlatformConfigProperty(String property) {
```

Comment:
Allows us to get properties from the PlatformConfig class (read form build.yml)
E.g. "x86_64-win32" will take precedence over the "common" platform.